ci/deps/actions-38.yaml

```diff
@@ -1,20 +1,50 @@
 name: pandas-dev
 channels:
-  - defaults
   - conda-forge
 dependencies:
   - python=3.8
 
-  # tools
-  - cython>=0.29.24
+  # test dependencies
```

Comment:
it *may* pay to have a single script able to generate *all* of these configs rather than hand editing e.g.

take min-versions and python versions that want to support and generate all config .yamls

Reply:
Agreed, want to head in this direction. If we can get the other builds on GHA instead of Azure (which have longer runtime allotments) we can just use a GHA Matrix to generate these dependencies.